src/AtmosphereRenderer.hpp

```diff
@@ -35,7 +35,7 @@ namespace csp::atmospheres {
 /// very same position as your planet. Set its scale to the same size as your planet.
 class AtmosphereRenderer : public IVistaOpenGLDraw {
  public:
-  AtmosphereRenderer(std::shared_ptr<Plugin::Properties> const& pProperties);
+  AtmosphereRenderer(std::shared_ptr<Plugin::Settings> const& pSettings);
```

Comment:
```suggestion
  AtmosphereRenderer(std::shared_ptr<Plugin::Settings> pSettings);
```

Reply:
You're right but in order to avoid too many conflicts with the `feature/clang-tidy` I will do this after it got merged.